app/controllers/api/v1/locations_controller.rb

```diff
@@ -27,9 +27,25 @@ def create
 
   def update
     location_to_update = Location.find(params[:id])
+    user = current_user.id
     new_rating = params[:_json]
-    location_to_update.rating += new_rating.to_i
-    if location_to_update.save
+
+    submit_rating = Rating.new(
+      user_id: user,
```

Comment:
no need to make this change, but just an fyi: thanks to ActiveRecord magic, you can do 
```
submit_rating = Rating.new(
      user: current_user,
      location: location_to_update,
...
```